week-3/refactor/format-as-12-hours.js

```diff
@@ -3,4 +3,43 @@
 // That implementation currently uses the expression Number(time.slice(0, 2)) twice
 // Store this expression in a variable and reference it twice in the function in the correct place
 
-// Explain why it makes more sense to store this expression in a variable
+// Explain why it makes more sense to store this expression in a variable - can be used frequently in other places, makes it look more neat 
+
+
+function formatAs12HourClock(time) {
+    let timeFormat = (Number(time.slice(0, 2)) - 12) 
```

Comment:
if you pass `08:00` in here, will timeFormat not become `-4`?

Reply:
I don't think so. All my assertion criteria pass